App.js

```diff
@@ -2,27 +2,49 @@ import { NavigationContainer } from '@react-navigation/native';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
 import 'expo-dev-client';
 import { RealmProvider } from '@realm/react';
-import React from 'react';
-// import bluetoothReceiver from './src/services/BluetoothReceiver';
+import React, { Component } from 'react';
 import Home from './src/pages/Home';
 import Settings from './src/pages/Settings';
 import Emergency from './src/pages/Emergency';
+import Dev from './src/pages/Dev';
+import { setupNotifications } from './src/services/notifications';
+import realmConfig from './src/services/db.js'
+import { LogBox } from 'react-native';
+import bluetoothReceiver from './src/services/bluetoothReceiver';
+
+LogBox.ignoreLogs(['`new NativeEventEmitter()`']); // Ignore log notification by message
 
 const Stack = createNativeStackNavigator();
 
-export default function App() {
+export default class App extends Component {
   // TODO - ask user for location permissions (otherwise bluetooth fails)
   // TODO - bluetooth here or on home screen?
 
-  return (
-    <RealmProvider>
-      <NavigationContainer>
-        <Stack.Navigator initialRouteName="Home">
-          <Stack.Screen name="Home" component={Home} />
-          <Stack.Screen name="Emergency" component={Emergency} />
-          <Stack.Screen name="Settings" component={Settings} />
-        </Stack.Navigator>
-      </NavigationContainer>
-    </RealmProvider>
-  );
+  componentDidMount() {
```

Comment:
I think we may be able to remove this, as I was just trying this out for testing. Let me double check though.

Or at least make App back into a function and not a class because I've heard the class pattern is bad practice for components

Reply:
made App back to a function